src/Tests/TemplateTests.cs

```diff
@@ -11,10 +11,11 @@
 [TestFixture]
 public class TemplateTests : IDisposable
 {
-    public TemplateTests()
+    [SetUp]
```

Comment:
Shouldn't this be `OneTimeSetUp` instead? I believe this should only happen once per test run.

Reply:
Ah, I guess that's what's causing the test failures. Will change.